raiden/network/discovery.py

```diff
@@ -45,7 +45,8 @@ class ContractDiscovery(Discovery):
     """On chain smart contract raiden node discovery: allows to register endpoints (host, port) for
     your ethereum-/raiden-address and looking up endpoints for other ethereum-/raiden-addressess.
     """
-    def __init__(self, blockchainservice, discovery_contract_address, poll_timeout=DEFAULT_POLL_TIMEOUT):
+    def __init__(self, blockchainservice, discovery_contract_address,
```

Comment:
Oh I am fine with placing the closing parentheses after the last argument. What I really want to emphasize is each argument on its own line if you are to break any arguments.

But note that flake8 accepts both closing parentheses styles. I don't have any strong opinion on that. Whatever you prefer.